### PR TITLE
Update contributing instructions

### PR DESCRIPTION
**Pull request by lisajian**
Going through the contributing instructions for the first time and added a bit more info for future contributors

* Added setup instructions
* Removed `--only` flag from running the dev server
* Update firebase.json with initialization of all emulators

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,12 +21,28 @@ We welcome any issues and Pull Requests to improve the Emulator UI. The followin
...
+
+```bash
+git clone git@github.com:firebase/firebase-tools-ui.git
+cd firebase-tools
+npm install # must be run the first time you clone
+```
+
 ### Start the Development Server
 
+Select the emulators that you will be interacting with in the UI by executing the command below and following the inline prompts:
```
Comment (yuchenshi) [inline README.md:36]: Please do this and commit the `firebase.json` changes to this PR so contributors don't have to do it again

Comment (lisajian) [reply README.md:36]: Sg, added!

Comment (yuchenshi) [inline README.md:36]: nit: Now you can remove this paragraph and `init` since it's already done

**Review by yuchenshi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lisajian) [reply README.md]: Ah whoops! Thanks for catching that!

**Pull request merged**